packages/vega-typings/tslint.json

```diff
@@ -6,6 +6,7 @@
     "max-line-length": false,
     "semicolon": false,
     "whitespace": false,
-    "file-name-casing": false
+    "file-name-casing": false,
+    "no-irregular-whitespace": false
```

Comment:
Why?

Reply:
In `packages/vega-typings/tests/dataflow/bar-time.ts` on line 2310 it has:

```
number: { decimal: ',', thousands: '.', grouping: [3], currency: ['', ' €'] },
```

The second currency raises an error with dtslint:

```
Error: /Users/saul/p/vega/packages/vega-typings/tests/dataflow/bar-time.ts:2310:76
ERROR: 2310:76  no-irregular-whitespace  Irregular whitespace not allowed

    at /Users/saul/p/vega/node_modules/dtslint/bin/index.js:207:19
    at Generator.next (<anonymous>)
    at fulfilled (/Users/saul/p/vega/node_modules/dtslint/bin/index.js:6:58)
error Command failed with exit code 1.
```

Reply:
Hmm, interesting. Oh well, we need to replace it with eslint eventually anyway.